src/main/kotlin/com/github/jensim/megamanipulator/actions/git/commit/CommitOperator.kt

```diff
@@ -20,7 +20,7 @@ class CommitOperator(
     private val localRepoOperator: LocalRepoOperator,
     private val processOperator: ProcessOperator,
     private val prRouter: PrRouter,
-    private val uiProtector: UiProtector,
+    private val uiProtector: UiProtector
```

Comment:
I love trailing commas, why remove it? :-D

Reply:
too much json in my head haha